### DISCUSSION TITLE
ssh -T git@mygogs.com 始终无法通过

### DISCUSSION THREAD

**Discussion by chuanxinwong** [gogs/gogs#6872] [中文讨论]
ssh -T git@mygogs.com ; 
提示如下:
Hi there, You've successfully authenticated, but Gogs does not provide shell access.
If this is unexpected, please log in with password and setup Gogs under another user.

我也配置了 .ssh/config

```
#github 
Host github.com
Hostname github.com
IdentityFile ~/.ssh/github_book

# gitee
Host gitee.com
HostName gitee.com
PreferredAuthentications publickey
IdentityFile ~/.ssh/rsa_gitee

# mygogs
Host mygogs.com
HostName xx.xx.xx.xxx   # 这里是使用的ip
PreferredAuthentications publickey
IdentityFile ~/.ssh/rsa_gogs_u1
IdentitiesOnly yes
```

gogs 个人设置里面 秘钥也配置了;

还是跟上面一样的提示 , 是还需要加什么配置吗?

**Comment by unknwon** [comment]
这提示的内容不是说了为什么了吗？有什么问题？

**Reply by chuanxinwong** [reply]
我确实不理解这个提示什么意思, 是我的服务器配置的有问题还是客户端有问题, 我在网上搜索了很多这个问题的回答, 全都是让我配置 .ssh/config , 但我觉得我这本地的配置没有问题呀,  我按照官网的说明 都重新安装三次了, 还是说有什么步骤漏掉了?

**Reply by unknwon** [reply]
https://www.deepl.com/en/translator#en/zh/Hi%20there%2C%20You've%20successfully%20authenticated%2C%20but%20Gogs%20does%20not%20provide%20shell%20access.%0AIf%20this%20is%20unexpected%2C%20please%20log%20in%20with%20password%20and%20setup%20Gogs%20under%20another%20user.

**Reply by chuanxinwong** [reply]
密码登录并在另一个用户下设置Gogs, 是在linux上登录另外一个账户, 还是在浏览器登录gogs的另外一个账户? 要修改哪些选项?

**Reply by unknwon** [reply]
Linux，如果你要进 git，先 ssh 到其它用户再 `su - git`.

**Reply by ruokid** [reply]
git用户设置了 nologin shell，是一个不能用于登录的用户

**Comment by Synex93** [comment]
老哥解决了 我好像也遇到了类似的问题
